src/intTest/resources/edifact/screening.edifact.dat

```diff
@@ -0,0 +1,43 @@
+UNA:+.? '
+UNB+UNOC:3+000000182400005:80+000000183500005:80+091113:1519+34++MEDRPT++1'
+UNH+1+MEDRPT:0:1:RT:NHS004'
+BGM+LSR'
+DTM+137:200911131519:203'
+S01+01'
+NAD+PO+++BCSP Hub Manager'
+RFF+AHI:UNKNOWN'
+SPR+PRO'
+S01+01'
+NAD+MR+++Bowel Cancer Screening Administrator'
+RFF+AHI:UNKNOWN'
+SPR+PRO'
+S01+01'
+NAD+MR+M87613:901'
+SPR+ORG'
```

Comment:
These segments also do not seem to be represented in the output FHIR JSON. However, it looks like something similar may be happening with Pathology, so this may require a separate ticket to investigate.

Reply:
I think we decided that we weren't going to map the `NAD+MR` segments because there's no equivalent FHIR JSON resource for `Message recipient` which is what `NAD+MR` is

Reply:
In **_pathology.edifact.dat_**, there's the following segments for NAD+MR:

```
S01+01'
NAD+MR+G3380314:900++SCOTT'
SPR+PRO'
S01+01'
NAD+MR+D82015:901'
SPR+ORG'
```

In **_pathology.fhir.json_**, these segments are mapped

```
S01+01'
NAD+MR+G3380314:900++SCOTT'
SPR+PRO'
```

but these segments are not:

```
S01+01'
NAD+MR+D82015:901'
SPR+ORG'
```



In **_screening.edifact.dat_**, there's the following segments:

```
S01+01'
NAD+MR+++Bowel Cancer Screening Administrator'
RFF+AHI:UNKNOWN'
SPR+PRO'
S01+01'
NAD+MR+M87613:901'
SPR+ORG'
```

However, none of these seem to be mapped in **_screening.fhir.json_**.

Reply:
In **pathology.edifact.dat**, these two segment groups contain the same information (`G3380314:900++SCOTT'`) and we're mapping the second one (through the `PractitionerMapper` which is why it appears as a `Practitioner` FHIR resource.

```
S01+01'
NAD+MR+G3380314:900++SCOTT'
SPR+PRO'
```

```
S01+01'
NAD+PO+G3380314:900++SCOTT'
SPR+PRO'
```

Reply:
Ah, I see. My mistake, thank you!

Reply:
Yep exactly. Details about this (and other decisions made about the adaptor) are documented here, https://gpitbjss.atlassian.net/wiki/spaces/NIA/pages/10865934377/P+S+design+decisions